docs/UserGuide.md

```diff
@@ -342,3 +315,96 @@ Action | Format, Examples
 **Undo** | `undo`
 **Help** | `help`
 **Exit** | `exit`
+
+## List of acceptable values
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
+.tg .tg-fymr{border-color:inherit;font-weight:bold;text-align:left;vertical-align:top}
+</style>
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-0pky"><span style="font-weight:bold">Command</span></th>
+    <th class="tg-fymr">Field</th>
+    <th class="tg-0pky"><span style="font-weight:bold">Acceptable values</span></th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0pky" rowspan="6"><code>createMember</code> / <code>editMember</code></td>
+    <td class="tg-0pky">MEMBER_NAME</td>
+    <td class="tg-0pky">Only contain alphanumeric characters and spaces, and should not be blank</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">GENDER</td>
+    <td class="tg-0pky"><code>Male</code>, <code>Female</code>, <code>Others</code></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">PHONE_NUMBER</td>
+    <td class="tg-0pky">Only contain numbers, and <span style="font-weight:bold">at least 3 digits long</span></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">EMAIL</td>
+    <td class="tg-0pky">A string with an email extension (e.g. <code>@gmail.com</code>)</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">ADDRESS</td>
+    <td class="tg-0pky">Any non-blank values except the following prefixes: <code>n/</code>, <code>g/</code>, <code>p/</code>, <code>e/</code>, <code>a/</code>, <code>t/</code></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">TAG</td>
+    <td class="tg-0pky">Only contain alphanumeric characters with no spaces in between, but can be blank</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky" rowspan="4"><code>createEvent</code> / <code>editEvent</code></td>
+    <td class="tg-0pky">EVENT_NAME</td>
+    <td class="tg-0pky">Only contain alphanumeric characters and spaces, and should not be blank</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">LOCATION</td>
+    <td class="tg-0pky"><span style="font-weight:bold">Must not be blank </span>and can take in any values, except the following prefixes:<code>e/</code>, <code>l/</code>, <code>d/</code>, <code>t/</code></td>
+  </tr>
+  <tr>
+    <td class="tg-0pky">DATE</td>
+    <td class="tg-0pky">Must be a <span style="font-weight:bold">valid date </span>in the format of <span style="font-weight:bold;font-style:italic">Y</span><span style="font-weight:bold">YYY-MM-DD</span></td>
```

Comment:
eh why is one Y italic here but the other 3 not lol :')

Reply:
Oops, an accident, will update